requirements.txt

```diff
@@ -6,4 +6,11 @@ tokenizers
 click
 jinja2
 certifi
-appdirs
\ No newline at end of file
+appdirs
+backoff
+boto3
+uvicorn
+fastapi
+tomli
+appdirs
+tomli-w
```

Comment:
The requirements file looks like it's for docker images, and I want our docker images to contain as many packages as possible so that users don't need to repackage images to install new dependencies